tests/server/urls.py

```diff
@@ -4,4 +4,8 @@
 
 
 admin.autodiscover()
-urlpatterns = [re_path(r"^admin/", admin.site.urls), re_path(r"^gcp/", include(django_gcp_urls))]
+urlpatterns = [
+    re_path(r"^admin/", admin.site.urls),
+    re_path(r"^gcp/", include(django_gcp_urls)),
+    re_path(r"^integrations/octue/", include("django_twined.urls")),
```

Comment:
Why is the include a string here but a module on the other includes?

Reply:
You can do it either way. I find it better not to import things if not necessary as it reduces the likelihood of circular imports

Reply:
OK, I don't mind which way of doing it, but I don't thing the include() is lazy so a circular import should manifest either way. Mostly I think the pattern should be consistent across the application

So you're welcome to either change the others or make this one a direct import, but consistent behaviour is neater :)